test cases/common/33 run program/meson.build

```diff
@@ -1,4 +1,4 @@
-project('run command', version : run_command('get-version.py', check : true).stdout().strip())
+project('run command', version : run_command('get-version.py', check : true).stdout().strip(), meson_version: '>=0.1.0')
```

Comment:
I think you dropped test.json in the rebase.

Reply:
Oops, yes. Fixed.

git range-diff considered the before/after versions of this series, to be so different it didn't want to provide diffs.